pkgs/development/tools/database/trino-cli/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchurl, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  version = "387";
```

Comment:
pname goes first, and after that is version

Reply:
gotcha 👍 , are these guidelines documented somewhere?